app/src/main/java/com/bitchat/android/onboarding/PermissionExplanationScreen.kt

```diff
@@ -237,20 +239,22 @@ private fun PermissionCategoryCard(
     }
 }
 
+@Composable
 private fun getPermissionEmoji(categoryName: String): String {
     return when (categoryName) {
-        "Nearby Devices" -> "📱"
```

Comment:
This fails on right to left languages for example.

Reply:
Can you explain how. This is just a string comparison.